scrape/scrape.go

```diff
@@ -1272,7 +1295,7 @@ loop:
 		ce, ok := sl.cache.get(yoloString(met))
 
 		if ok {
-			err = app.AddFast(ce.ref, t, v)
+			err = app.AddFast(ce.ref, ce.meta, t, v)
```

Comment:
What if this is from a previous scrape, and this scrape doesn't have metadata?

Reply:
I was assuming perhaps that it would keep the metadata it previously declared? Or should it "zero" the metadata instead?